server/src/main/java/org/opensearch/cluster/decommission/DecommissionService.java

```diff
@@ -481,4 +482,51 @@ public void onFailure(Exception e) {
             }
         };
     }
+
+    public void deleteDecommissionAttribute(final ActionListener<AcknowledgedResponse> listener) {
+        decommissionController.clearVotingConfigExclusion(new ActionListener<Void>() {
+            @Override
+            public void onResponse(Void unused) {
+                logger.info("successfully cleared voting config exclusion for delting the decommission");
+                clusterUpdateTaskForDeletingDecommission(listener);
```

Comment:
I'd prefer to name this - `deleteDecommissionState`

Reply:
Done.